### DISCUSSION TITLE
Integrate Oqtane with Grafana & Prometheus

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#4462] [General] [answered]
Hi,

Any plans to Integrate Oqtane with  [Grafana](https://grafana.com/) & Prometheus ?

**Comment by sbwalker** [comment, marked as answer]
@Sharafudeen I think what you are asking is if there is a plan to integrate OpenTelemetry (https://opentelemetry.io/). The new .NET Aspire project from Microsoft is using OpenTelemetry for observability (https://github.com/open-telemetry/opentelemetry-dotnet). The OpenTelemetry data can be consumed by observability tools like Grafana or Prometheus. 

At this point OpenTelemetry integration support is not a priority in comparison to other features. However you could certainly use it in your own custom modules (that is the benefit of Oqtane's extensibility model). In the the future it could be a complementary feature to the scale out capability which is scheduled for Oqtane in Q4 (.NET 9)

**Reply by Sharafudeen** [reply]
@sbwalker Thanks for the information.
